packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/web_resource_error.dart

```diff
@@ -120,4 +121,7 @@ class WebResourceError {
 
   /// Whether the error originated from the main frame.
   final bool? isForMainFrame;
+
+  /// The URL for which the failing resource request was made.
+  final String? failingUrl;
```

Comment:
Should we just call it `url`? The fact that it's failing seems like it's inherent in this being part of an error structure.

Or maybe `requestUrl`?

Reply:
`url` seems better to me too. However, Android already has a `failingUrl`. I think it would be easier to keep `failingUrl` to avoid having 2 values that are equal on Android?

Reply:
We could just change `failingUrl` to be a deprecated alias of `url` once this lands.